<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lFeedback">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chDevMode">
     <property name="text">
      <string>Enable developer mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="localGit">
      <attribute name="title">
       <string>Git local</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="globalGit">
      <attribute name="title">
       <string>Git global</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <attribute name="title">
       <string>GitQlient</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Disable logs</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="chDisableLogs">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Commit title max. length</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="spCommitTitleLength">
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Set log level</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="cbLogLevel">
            <item>
             <property name="text">
              <string>Trace</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Debug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Info</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warning</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Error</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fatal</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Styles schema</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="cbStyle">
            <item>
             <property name="text">
              <string>dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bright</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Git Location (if not in PATH)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" colspan="3">
           <widget class="QLineEdit" name="leGitPath">
            <property name="placeholderText">
             <string>Git location...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Repository</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>551</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_2">
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <property name="spacing">
               <number>10</number>
              </property>
              <item row="9" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Enable pomodoro</string>
                </property>
               </widget>
              </item>
              <item row="12" column="1">
               <widget class="QCheckBox" name="cbSubtree">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QCheckBox" name="cbStash">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Update submodules when pull</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Show submodules</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Order of commits in the graph</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Auto-Fetch interval</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="autoFetch">
                <property name="suffix">
                 <string> minutes</string>
                </property>
                <property name="maximum">
                 <number>60</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QCheckBox" name="cbSubmodule">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QCheckBox" name="updateOnPull">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="sbMaxCommits">
                <property name="suffix">
                 <string> last commits</string>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Language</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Max. commits to retrieve (0 for all)</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QCheckBox" name="cbPomodoroEnabled">
                <property name="text">
                 <string/>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="lCacheSize">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="ButtonLink" name="pbClearCache">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>The interval is expected to be in minutes. Choose a value between 0 (for disabled) and 60.</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Prune when fetch</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbTranslations"/>
              </item>
              <item row="7" column="1">
               <widget class="QCheckBox" name="clangFormat">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelClangFormat">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run clang-format before commit&lt;br/&gt;(Requires clang-format)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="12" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Show subtrees</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Cache space used:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cbLogOrder">
                <item>
                 <property name="text">
                  <string>Author date order</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Date order</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Topo order</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="pruneOnFetch">
                <property name="text">
                 <string/>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Show stashes</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <attribute name="title">
       <string>Build System</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leBsUser"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="leBsUserLabel">
            <property name="text">
             <string>User:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leBsUrl"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="leBsUrlLabel">
            <property name="text">
             <string>Endpoint URL:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chBoxBuildSystem">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="leBsTokenLabel">
            <property name="text">
             <string>Token:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Enable build system view</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leBsToken">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonLink</class>
   <extends>QLabel</extends>
   <header>ButtonLink.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chDisableLogs</tabstop>
  <tabstop>cbLogLevel</tabstop>
  <tabstop>leGitPath</tabstop>
  <tabstop>cbTranslations</tabstop>
  <tabstop>sbMaxCommits</tabstop>
  <tabstop>cbLogOrder</tabstop>
  <tabstop>autoFetch</tabstop>
  <tabstop>pruneOnFetch</tabstop>
  <tabstop>updateOnPull</tabstop>
  <tabstop>clangFormat</tabstop>
  <tabstop>cbPomodoroEnabled</tabstop>
  <tabstop>chBoxBuildSystem</tabstop>
  <tabstop>leBsUrl</tabstop>
  <tabstop>leBsUser</tabstop>
  <tabstop>leBsToken</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>saveConfig()</slot>
 </slots>
</ui>
